### ISSUE TITLE
Opening doors non-vanilla building using ACE3 key shortcut.

### ISSUE THREAD

**Issue by ZikiCZ** [acemod/ACE3#5199]
**Arma 3 Version:** `1.70+hotfix` (stable)
**CBA Version:** `3.3.1` (stable)
**ACE3 Version:**`3.7.2` (stable)

**Mods:**
```
- CBA_A3
- ace
-acex
-CUP terrains (Core+Maps+CWA) (ver.: 1.3.0 (stable))
```

**Description:**
- I'd like to report the broken functionality of opening doors through ACE key shortcut in relation to a recommendation from Jonpas.
 In the video listen bellow, I compare the difference between the vanilla and Apex buildings (where everything works as it should) and CUP. Two strange things happened with CUP, after pressing the key shortcut, the sound can be heard, but the doors still won't open (0:30; 0:53). In the second case, even the sound can not be heard (0:40).
We further know that opening doors with the shortcut doesn't work in other mods e.g.: Open Chernarus and maps with their custom buildings such as Isla Duala.
https://youtu.be/0BA754E_27s

**Steps to reproduce:**
- Load any CUP map and try to open door at any non-vanilla building.

**Where did the issue occur?**
- Dedicated / Self-Hosted Multiplayer / Singleplayer / Editor (Singleplayer) / Editor (Multiplayer)

**Placed Modules:**
- Used defualt ace settings. No additional modules.

**RPT log file:**
- RTP: 
[Arma3_x64_2017-05-25_15-12-25.txt](https://github.com/acemod/ACE3/files/1029078/Arma3_x64_2017-05-25_15-12-25.txt)
-ACE_debug:
[ace_debug.txt](https://github.com/acemod/ACE3/files/1029075/ace_debug.txt)

**Comment by Phyma** [comment]
Checking the CUP house problem and the biggest problem i can find is with their configs. Because vanilla  houses  have a class SimpleObject with a property `animate[] = {["door_1_rot",0],["door_1_handle_rot_1",0],["door_1_handle_rot_2",0],["door_1_locked_rot",0],["door_1_handle_locked_rot",0],["door_2_rot",0],["door_2_handle_rot_1",0],["door_2_handle_rot_2",0],["door_2_locked_rot",0],["door_2_handle_locked_rot",0],["glass_1_hide",0],["glass_1_unhide",0],["glass_2_hide",0],["glass_2_unhide",0],["glass_3_hide",0],["glass_3_unhide",0],["glass_4_hide",0],["glass_4_unhide",0]};`

But on CUP houses its empty just animate[] = {};
So animate  dosent work. It tryes to set the value but nothing happens because animate cmd dosent work.

So is this a bug with CUP? and not ACE3

**Comment by Phyma** [comment]
Possible fix in: #5226

**Comment by Phyma** [comment]
This is merged please close

**Comment by PabstMirror** [comment]
Closed via #5226 - Thanks @Phyma
